core/templates/components/state-editor/state-interaction-editor/state-interaction-editor.component.ts

```diff
@@ -77,6 +77,7 @@ export class StateInteractionEditorComponent
   directiveSubscriptions = new Subscription();
   hasLoaded: boolean;
   interactionEditorIsShown: boolean;
+  showToggleBtn: boolean = false;
```

Comment:
Do we need to initialize with a `false` value if we already do it in the `ngOnInit` function?

Reply:
no we don't need it but just did it to make it better but I think this arises no issues and as the button remains false everywhere so I don't think that there is any need for adding a test.
So, I hope everything is fine. I will remove the initialisation.

Reply:
@AdityaRaimec22 There are two user interactions with the toggle button, .i.e, toggle and untoggle. To catch such regressions, we must cover all possible user interactions in the unit tests. Hence, we need to add another unit test.

Reply:
PTAL on below comments.